portfolio/src/main/webapp/calendar.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Models a calendar event that is displayed on the UI.
+ * This class is useful when the event information needs to be sent
+ * to the calendar servlet through a POST request.
+ * This class contains all the event information that can be easily
+ * converted into a JSON string.
+ * All fields are required.
+ * @param name: of type String, name of event
```

Comment:
The @param comment can be a method comment on the constructor, rather than a class level comment.

Example:
https://google.github.io/styleguide/jsguide.html#jsdoc-tags

Reply:
And let's have one @param for each parameter.